<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabQFile">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/images/804.bmp</normaloff>:/images/images/804.bmp</iconset>
       </attribute>
       <attribute name="title">
        <string>QFile直接操作</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="textQFile"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabQTextStream">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/images/806.bmp</normaloff>:/images/images/806.bmp</iconset>
       </attribute>
       <attribute name="title">
        <string>QTextStream操作</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPlainTextEdit" name="textQTextStream"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQFileOpen"/>
   <addaction name="actionQFileSaveAs"/>
   <addaction name="actionQSaveFileSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionQTextStreamOpen"/>
   <addaction name="actionQTextStreamSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionQFileOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/804.bmp</normaloff>:/images/images/804.bmp</iconset>
   </property>
   <property name="text">
    <string>QFile打开</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionQFileSaveAs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/104.bmp</normaloff>:/images/images/104.bmp</iconset>
   </property>
   <property name="text">
    <string>QFile另存</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionQSaveFileSaveAs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/066.GIF</normaloff>:/images/images/066.GIF</iconset>
   </property>
   <property name="text">
    <string>QSaveFile另存</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionQTextStreamOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/122.bmp</normaloff>:/images/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>QTextStream打开</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionQTextStreamSaveAs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/007.GIF</normaloff>:/images/images/007.GIF</iconset>
   </property>
   <property name="text">
    <string>QTextStream另存</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
